### PR TITLE
Adding fstrm to SPEC Extended

### PR DESCRIPTION
**Pull request by AZaugg**
Adding a new SPEC-EXTENDED package to Mariner - fstrm

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Adding a new package(fstrm) to SPEC-Extended

###### Change Log  

- Change: Adding new package fstrm

###### Does this affect the toolchain?  

NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

**Review by Camelron — APPROVED** [review]

File: SPECS-EXTENDED/fstrm/fstrm.signatures.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "fstrm-v0.6.1.tar.gzz": "4f0f7ad2b760119c441aba58271e84de683b3cc138530d02710896641866e2d2"
```
Comment (Camelron) [inline SPECS-EXTENDED/fstrm/fstrm.signatures.json:3]: Small typo, please remove extra z in gz

I get a different hash:
bca4ac1e982a2d923ccd24cce2c98f4ceeed5009694430f73fc0dcebca8f098f  fstrm-0.6.1.tar.gz

**Review by Camelron — CHANGES_REQUESTED** [review]

**Review by Camelron — APPROVED** [review]

**Review by rlmenge — CHANGES_REQUESTED** [review]
Missing license info in SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

See https://github.com/microsoft/CBL-Mariner/pull/4841 for reference

**Review by rlmenge — APPROVED** [review]

File: SPECS-EXTENDED/fstrm/fstrm.spec [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+%{_includedir}/fstrm/
+%{_libdir}/pkgconfig/libfstrm.pc
+%{_libdir}/libfstrm.so
+
+%files doc
+%doc %{_pkgdocdir}/html
+
+%changelog
+* Thu Feb 23 2023 Andy Zaugg <azaugg@linkedin.com> - 0.6.1-1
+- First version of fstrm for cm2 Spec file imported from Centos EPEL
```
Comment (rlmenge) [inline SPECS-EXTENDED/fstrm/fstrm.spec:89]: In order to pass our license checks, you need to specifiy to license of the distro in the changelog. It should match the format specified here: https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/scripts/spec_source_attributions.py

Comment (AZaugg) [reply SPECS-EXTENDED/fstrm/fstrm.spec]: OK, I think I got it now 😄

File: SPECS-EXTENDED/fstrm/fstrm.spec [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+%{_includedir}/fstrm/
+%{_libdir}/pkgconfig/libfstrm.pc
+%{_libdir}/libfstrm.so
+
+%files doc
+%doc %{_pkgdocdir}/html
+
+%changelog
+* Thu Feb 23 2023 Andy Zaugg <azaugg@linkedin.com> - 0.6.1-1
+- Initial CBL-Mariner import from CentOS 8 (license: MIT).
```
Comment (rlmenge) [inline SPECS-EXTENDED/fstrm/fstrm.spec:89]: Please add back the 
`- License verified`

**Review by rlmenge — CHANGES_REQUESTED** [review]
Please add back the entry for SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md

File: SPECS-EXTENDED/fstrm/fstrm.spec [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- License verfied
```
Comment (rlmenge) [inline SPECS-EXTENDED/fstrm/fstrm.spec:90]:
```suggestion
- License verified
```

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json [hunk 5]
```diff
@@ -242,6 +242,7 @@
                 "fribidi",
                 "fros",
                 "frr",
+                "fstrm",
```
Comment (rlmenge) [inline SPECS/LICENSES-AND-NOTICES/data/licenses.json:245]:
```suggestion
```

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json
Comment (rlmenge) [inline SPECS/LICENSES-AND-NOTICES/data/licenses.json:1]: Please move fstrm to the CentOS section

File: SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md
Comment (rlmenge) [inline SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md:1]: Please move fstrm to the CentOS section

**Review by rlmenge — CHANGES_REQUESTED** [review]
Please move fstrm to the CentOS spots in the license files and fix misspelling to apease the license checks :)

**Review by rlmenge — APPROVED** [review]

**Pull request merged**